src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -4923,6 +5134,9 @@ namespace HVACVariableRefrigerantFlow {
             MyOneTimeFlag = false;
             MyVRFCondFlag = true;
 
+            MySuppCoilPlantScanFlag.allocate(NumVRFTU);
+            MySuppCoilPlantScanFlag = true;
```

Comment:
Seems like you'd want to just add a boolean, defaulted to true, to the VRFTU structure, so that you don't have a separate array to manage.  If there are any other changes to make here, I'd like to see that streamlined, or maybe you have a specific reason you needed to do it this way.

Reply:
No, I did not have any specific reason. I am happy to change it along with your next comment about the nested IF structure. Let me know.